client/src/components/isLoggedIn.js

```diff
@@ -0,0 +1,7 @@
+export const isLoggedIn = () => {
```

Comment:
Great 👍 Just this file could be moved in another folder as it is more a utility function rather than a component. How about moving this file under a new folder: `client/src/helpers/isLoggedIn.js` ?